plugins/inputs/http_listener_v2/sample.conf

```diff
@@ -3,6 +3,17 @@
   ## Address and port to host HTTP listener on
   service_address = ":8080"
 
+  ## Network type to listen to default tcp
+  # if set to unix service_address will be interpreted as unix socket path
+  # network = "tcp"
```

Comment:
How about extending the `service_address` to be able to take an URL? E.g. `service_address = ":8080"` or `service_address = "tcp://127.0.0.1:8080"` or `service_address = "unix:///var/spool/mysocket"`? If no schema can be recognized we assume `tcp`... What do you think?

Reply:
hi Yes that could be done I guess this will also make the configuration reflect what is in the socket_listener so it makes sense.